converter/parser.h

```diff
@@ -2,20 +2,32 @@
 #define PARSER_H
 
 
-#include <fstream>
 #include <iostream>
```

Comment:
Why do you need iostream here?

Reply:
Yes, it is useless here

Reply:
please update the code and update Pull Request.